app/models/concept-scheme.js

```diff
@@ -0,0 +1,11 @@
+import Model, { attr, hasMany } from '@ember-data/model';
+
+export default class ConceptSchemeModel extends Model {
+  @attr uri;
+  @attr preflabel;
+  @hasMany('concept', { inverse: null, async: true }) concepts;
+
+  get label() {
+    return this.preflabel;
```

Comment:
In the backend I have set this to `prefLabel` but for some treason it is using `preflabel` with a lower `L`. this is why im using the getter for the label here. => when the backend changes this is the place to change it and not in the "whole code base"